kohttp/src/main/kotlin/io/github/rybalkinsd/kohttp/interceptors/RetryInterceptor.kt

```diff
@@ -34,7 +35,13 @@ class RetryInterceptor(
                     delay = performAndReturnDelay(delay)
                 }
                 val response = chain.proceed(request)
-                if (!isRetry(response, attemptsCount)) return response
+
+                val nextTime = calculateNextRetry(response, attemptsCount, delay, chain.readTimeoutMillis())
```

Comment:
There is a room for improvement here.

`if () statement else return ` logic looks rather complicated